tests/Helpers/WebserverRunner.php

```diff
@@ -0,0 +1,54 @@
+<?php
```

Comment:
Should we just remove this?

Reply:
Yes, I was going to do more changes in another PR though.

Reply:
Oh did this never make it to `main` -- yeah okay I'll remove it